app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+    </style>
+  </head>
+
+  <body>
+   <h1> Post Info <h1/>
+   <%= @post.text %>
+    <p>Comments:
+    <% @post.query_comments.each do |comment| %>
```

Comment:
- [ ] I was unable to create a comment so I am unable to view this page in its entirety. Kindly look into your Comment migrations. There is no `post_id` in the table. Additionally, you have `user_id` and `author_id` which are redundant. From a previous project, you were supposed to have `author_id` and `post_id`.

  ![image](https://user-images.githubusercontent.com/36542986/225566995-29603203-8abb-4e9b-92ae-29a95c4aa15f.png)
  <hr/>
  
  ![image](https://user-images.githubusercontent.com/36542986/225571897-f07d43fb-6b20-4c65-9739-56b383f963cc.png)